### ISSUE TITLE
More than one file was found with OS independent path 'META-INF/library_release.kotlin_module'

### ISSUE THREAD

**Issue by AlbertVilaCalvo** [ChuckerTeam/chucker#134]
Hi, when I add the library to the `app/build.gradle` file:

```
debugImplementation "com.github.ChuckerTeam.Chucker:library:3.0.1"
releaseImplementation "com.github.ChuckerTeam.Chucker:library-no-op:3.0.1"
```
I get the following build error:

```
Execution failed for task ':app:mergeDebugJavaResource'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
   > More than one file was found with OS independent path 'META-INF/library_release.kotlin_module'
```

Indeed, I have 2 libraries with this file, one of them is chucker:

![Captura de Pantalla 2019-10-10 a les 12 58 32](https://user-images.githubusercontent.com/4823860/66563306-cf111580-eb5d-11e9-8d06-878e5d5dc903.png)

There is a way to fix this with `packagingOptions`:

```
packagingOptions {
    exclude 'META-INF/library_release.kotlin_module'
}
```

However it seems that the root cause can be solved on the library, which is better because it avoids having everyone to add the exclude fix:

https://github.com/Takhion/android-extras-delegates/issues/1
https://github.com/Kodein-Framework/Kodein-DI/issues/138

Would be nice if this was fixed at the library :)

**Comment by AlbertVilaCalvo** [comment]
I've found this related to the error: https://readyset.build/build-failures-kotlin-metadata-errors-f0090cdf948c

**Comment by AlbertVilaCalvo** [comment]
I've opened an issue on the other library too: https://github.com/kizitonwose/CalendarView/issues/65

**Comment by cortinico** [comment]
Thanks for the report @AlbertVilaCalvo 
Fix should be #146

**Comment by jhamin0511** [comment]
It is work for me.

    packagingOptions {
        exclude 'META-INF/library_release.kotlin_module'
    }

https://stackoverflow.com/questions/44509608/duplicate-files-copied-in-apk-meta-inf-library-release-kotlin-module
